recipes/deon/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "deon" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 509adbf731c91881e517b5b56dd9a54112997baf00e2dd340644ae37c5f3c453
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - beautifulsoup4 >=4.6.1
+    - click >=6.7
+    - pyyaml >=3.13
+
+test:
+  imports:
+    - deon
+  commands:
+    - deon --help
+
+about:
+  home: http://deon.drivendata.org/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'A command line tool to easily add an ethics checklist to your data science projects.'
+  description: |
+    deon is a command line tool that allows you to easily add an ethics
+    checklist to your data science projects. The conversation about ethics in
+    data science, machine learning, and AI is increasingly important. The goal
+    of deon is to push that conversation forward and provide concrete,
+    actionable reminders to the developers that have influence over how data
+    science gets done.
+  doc_url: http://deon.drivendata.org/
+  dev_url: https://github.com/drivendataorg/deon
+
+extra:
+  recipe-maintainers:
+    - ejm714
```

Comment:
@ejm714 Can you comment about if you're happy to be listed as a maintainer here?

Reply:
Yep happy to be a maintainer!